cachito/web/models.py

```diff
@@ -625,6 +636,10 @@ def _state_to_json(state):
             rv["packages"] = [
                 package.to_json(package_to_deps.get(package.id, [])) for package in self.packages
             ]
+            if current_app.config["CACHITO_REQUEST_FILE_LOGS_DIR"]:
+                rv["logs"] = {
+                    "url": url_for(".get_request_logs", request_id=self.id, _external=True)
```

Comment:
Should we use the full endpoint as in the other calls to `url_for` here? Like `api_v1.get_request_logs`?

Reply:
It doesn't matter as it provides the full link anyway but I can change that for the consistency!